src/containers/FormikForm/articleFormHooks.js

```diff
@@ -123,10 +123,6 @@ export function useArticleFormHooks({
       setSavedToServer(true);
       formik.resetForm({ values: getInitialValues(savedArticle) });
 
-      Object.keys(formik.values).map(fieldName =>
-        formik.setFieldTouched(fieldName, true, true),
-      );
-
```

Comment:
Det er denne som tar det meste av tiden under lagring. Er den nødvendig, @jnatten ? Ser den ble lagt til i https://github.com/NDLANO/editorial-frontend/pull/619, men lagring med validerings error fungerer fortsatt

Reply:
Jeg _tror_ den ikke er nødvendig etter https://github.com/NDLANO/editorial-frontend/pull/736
Oppførselen før var at vi ikke ville at validering skulle trigges på untouched felter før lagring, men nå så bryr vi oss ikke om feltet er touched eller ikke under validering i det hele tatt.

Dersom vi finner ut at det er nødvendig så kan vi sikkert spare tid ved å endre det siste parameteret (`shouldValidate`) til `false` og triggre validering manuelt etter alle feltene og ikke for hvert felt.

Reply:
Synes uansett validering til slutt høres ut som en bedre ide. Så tenker vi berre kan ta inn denne.